### PR TITLE
Ahodges/RESbumper

### PR DESCRIPTION
**Pull request by lamagnifica**
Work in Progress: events for pre-roll videos. 
See event spec here: https://openedx.atlassian.net/wiki/pages/viewpage.action?pageId=29689098
The PR for the course author's info is here: https://github.com/edx/edx-documentation/pull/344
@explorerleslie @stroilova @brianhw @auraz I have a few open questions on the spec, and you'll see "%%" in places where I know I need more information. I'll be able to work on this again starting on Thursday June 4.

### CODE REVIEW

**Comment by lamagnifica** [comment]
New text requiring review is in tracking_logs.rst, starting line 1268

**Comment by lamagnifica** [comment]
@explorerleslie @stroilova @brianhw @auraz ready for review, expected release is Tuesday

File: en_us/data/source/internal_data_formats/event_list.rst [hunk 1]
```diff
@@ -78,10 +78,29 @@ Alphabetical Event List
      - :ref:`content`
    * - ``edx.googlecomponent.document.displayed``
      - :ref:`content`
+<<<<<<< HEAD
```
Comment (polesye) [inline en_us/data/source/internal_data_formats/event_list.rst]: Should it be removed?

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 2]
```diff
@@ -1227,11 +1204,11 @@ selected either the same speed or a different speed.
      - integer
      - The time in the video that the user chose to change the playing speed. 
    * - ``new_speed``
-     - 
+     - integer
```
Comment (polesye) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: it sends a float value as a string.

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 3]
```diff
@@ -1227,11 +1204,11 @@ selected either the same speed or a different speed.
...
      - The speed that the user selected for the video to play: '0.75', '1.0',
        '1.25', '1.50'.
    * - ``old_speed``
-     - 
+     - integer
```
Comment (polesye) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: it sends a float value as a string.

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 4]
```diff
@@ -1283,9 +1260,272 @@ The ``stop_video``/``edx.video.stopped`` events include the following
...
+* ``code``
+* ``currentTime``: The point in the file at which the pre-roll video was
+  skipped.
+* ``duration``
+* ``host_component_id``
+
+``edx.video.bumper.stopped``
+****************************
+
+A browser emits this event when either a user selects the video player's **pause** control or when the video player reaches the end of the
```
Comment (polesye) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: There is no `pause` control.
I suggest to fix it so:
`A browser emits this event when the video player reaches the end of the ...`

File: en_us/data/source/internal_data_formats/event_list.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/data/source/internal_data_formats/event_list.rst]: thank you!

**Comment by explorerleslie** [comment]
:+1:

**Comment by polesye** [comment]
@auraz is on vacation, so I did review instead of him :+1:

**Pull request merged**